src/access.py

```diff
@@ -33,7 +42,22 @@ def approved_user():
 
   return approved_caller.split(':')[1] #return userid portion of tag
 
-def session_user(encoded_jwt):
+def store_to_ssm(access_token):
+  parameter_name = os.environ.get(ssm_parameter_name_env_var)
+  if not (parameter_name):
+    # just exit early if the parameter name to store in SSM is not found
+    return
+
+  ssm_client = boto3.client('ssm', region)
+
+  ssm_client.put_parameter(
```

Comment:
I'll be working on custom KMS key as the next step after verifying that I can get a working integration when just using the default.

Reply:
i believe we already have a key for SC related secrets.  https://github.com/Sage-Bionetworks-IT/organizations-infra/blob/ccf18e75a6f2c7df53fad5a20e1954687d9e1848/sceptre/scipool/config/develop/sc-kms-key.yaml

Reply:
From discussions with @ahayden, each instance would want its own key because we don't want 2 notebook instances being alloweded to retrieve each other's credentials.

The InstanceRoles currently have permission to GetParameter on any Parameter Store Resource (from ManagedPolicy: `arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore`)